### PR TITLE
Updated Polish (pl) localization of Appstores

### PR DESCRIPTION
**Pull request by piotrdrag**
@splewako

### CODE REVIEW

File: pl/android_42_release.lang [hunk 1]
```diff
@@ -174,57 +174,57 @@ Więcej informacji: {{mozilla_org_link}}
...
 # Screenshot text, displayed as a bold title
 ;Intelligent searches
-Intelligent searches
+Inteligentne wyszukiwanie
 
 
 # Screenshot description below the title, adjust line break tag (<br>) position for your language
 ;Get accurate, predictive results <br>before you finish typing
-Get accurate, predictive results <br>before you finish typing
+Dokładne podpowiedzi pojawiają się<br>jeszcze przed ukończeniem pisania
```
Comment (splewako) [inline pl/android_42_release.lang]: "Celne" maybe?

**Comment by splewako** [comment]
One comment above but feel free to land.

File: pl/android_42_release.lang (hunk 1 above)
Comment (piotrdrag) [inline pl/android_42_release.lang]: Sure, why not.

**Comment by piotrdrag** [comment]
Thanks!

**Pull request merged**